.github/workflows/pylint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r test-requirements.txt
```

Comment:
If all it's going to run after this is pylint, do we really want to install the entire requirements-dev?

Reply:
Yes, otherwise the pylint checks fail, example [here](https://github.com/chetanph/fmc-remediation-module-xdr/actions/runs/9845072877/job/27179802096)

Reply:
I see! Thanks!